certified-connectors/Zoho Mail/apiDefinition.swagger.json

```diff
@@ -1344,8 +1343,7 @@
                     "type": "object",
                     "properties": {
                       "labelId": {
-                        "type": "integer",
```

Comment:
The "type" has been changed from integer to string, can you please confirm that no existing flows or users will be affected by this change?

Reply:
yes, it won't affect the existing user.